e2e/actions/oci.go

```diff
@@ -172,6 +173,10 @@ func (c actionTests) actionOciExec(t *testing.T) {
 				e2e.ExpectOutput(e2e.RegexMatch, `\bHOME=/myhomeloc\b`),
 				e2e.ExpectOutput(e2e.RegexMatch, `\btmpfs on /myhomeloc\b`),
 			},
+			skipProfiles: map[string]bool{
```

Comment:
These should definitely work, but it's a separate issue from the problem that's fixed here, so we'll handle it separately.